protocol/slp/commit.md

```diff
@@ -0,0 +1,28 @@
+# COMMIT - Checksum Commitment Transaction
+
+As previously discussed, a token issuer should make regular commitments of the SHA-256 hash of previous transactions made for this token.
+Although this is not part of the consensus rules (commitments may occur outside of the token transaction graph, and commitment data is never used in consensus rules), it allows a user to verify that the issuer is accurately honoring the token's consensus rules.
+This increases confidence that tokens will be judged as expected at the time of redemption.
+
+Initial implementations will focus on supporting the consensus-based forms of validation, and so at this time the exact format for the commitment document is not specified.
+The general ideas will be:
```

Comment:
Maybe something more like "General expectations of a commitment format should include:"